usr/share/rear/layout/prepare/GNU/Linux/130_include_filesystem_code.sh

```diff
@@ -113,16 +113,37 @@ function create_fs () {
         (xfs)
             # If available use wipefs to cleanup disk partition:
             test "$has_wipefs" && echo "$wipefs_command" >> "$LAYOUT_CODE"
-            # Actually create the filesystem:
-            echo "mkfs.xfs -f $device" >> "$LAYOUT_CODE"
+
+            local opts=""
+            local set_uuid_using_xfs_admin=0
+
+            # Decide if mkfs.xfs or xfs_admin will set uuid.
+            # Uuid set by xfs_admin will set incompatible flag on systems with
+            # enabled CRC. This might cause ReaR failure during grub installation.
+            # See: https://github.com/rear/rear/issues/1065
+            if [ -n "$uuid" ] ; then
+                # Check if option '-m uuid=<uuid>' is available in mkfs.xfs
+                if [ $(mkfs.xfs --help 2>&1 | grep "\-m" | grep -c uuid) -ne 0 ]; then
```

Comment:
Well, there can be. But checking option this way, looks more accurate for me.
Except that, you can never be sure that version behavior will be same across distributions ...